openapi/rhose-api.yaml

```diff
@@ -10,6 +10,11 @@ info:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
   version: 0.0.1
+servers:
+- url: http://localhost:8080
```

Comment:
Quarkus now generates this value. We can override this with https://quarkus.io/guides/openapi-swaggerui#quarkus-smallrye-openapi_quarkus.smallrye-openapi.servers but we need to know what to address. It might be ok for the time being, if this is not going to break the client generated by @kelvah

Reply:
@r00ta I did a quick check and it changes the default base url for APIs calls. Not a big deal because it's not used. We set up a different base url based on the env.
FYI other services have [this configuration for servers](https://github.com/redhat-developer/app-services-sdk-js/blob/main/.openapi/connector_mgmt.yaml#L8).

Reply:
As far as I remember the list of `servers` must be "limited" to what we need  (aka api.openshift etc) so we should not include undesired servers

Reply:
This PR #861 from @jomarko adds also the list of servers. If we merge it before the bump it should be solved otherwise just merge this and he can rebase applying the change

Reply:
see `manager/src/main/resources/application.properties` in my #861